plugins/git/octo.nix

```diff
@@ -0,0 +1,169 @@
+            Copy url to system clipboard.
+          '';
+
+          checkout_pr = mkMappingOption "<C-o>" ''
+            Checkout pull request.
+          '';
+
+          merge_pr = mkMappingOption "<C-r>" ''
+            Merge pull request.
+          '';
+        };
+    };
+
+    reaction_viewer_hint_icon = helpers.defaultNullOpts.mkStr "" ''
+      Marker for user reactions.
+    '';
+
+    user_icon = helpers.defaultNullOpts.mkStr " " ''
+      User Icon.
+    '';
+
+    timeline_marker = helpers.defaultNullOpts.mkStr "" ''
+      Timeline marker.
+    '';
+
+    timeline_indent = helpers.defaultNullOpts.mkStr "2" ''
+      Timeline indentation.
+    '';
+
+    right_bubble_delimiter = helpers.defaultNullOpts.mkStr "" ''
+      Bubble delimiter.
+    '';
+
+    left_bubble_delimiter = helpers.defaultNullOpts.mkStr "" ''
+      Bubble delimiter.
+    '';
+
+    github_hostname = helpers.defaultNullOpts.mkStr "" ''
+      Github Enterprise host.
+    '';
+
+    snippet_context_lines = helpers.defaultNullOpts.mkInt 4 ''
+      Number of lines around commented lines.
+    '';
+
+    gh_env = helpers.defaultNullOpts.mkAttributeSet "{ }" ''
+      Extra environment variables to pass on to GitHub CLI, can be a table or function returning a table.
+    '';
+
+    timeout = helpers.defaultNullOpts.mkInt 5000 ''
+      Timeout for requests between the remote server.
+    '';
+
+    ui = helpers.mkCompositeOption "UI options." {
+      use_sign_column = helpers.defaultNullOpts.mkBool true ''
+        Show "modified" marks on the sign column.
+      '';
+    };
+
+    issues =
+      helpers.mkCompositeOption
+        ''
+          Issues options.
+        ''
+        {
+          order_by =
+            helpers.mkCompositeOption
+              ''
+                Criteria to sort results of `Octo issue list`.
+              ''
+              {
+                field =
+                  helpers.defaultNullOpts.mkEnum
+                    [
+                      "CREATED_AT"
+                      "COMMENTS"
+                      "UPDATED_AT"
+                    ]
+                    "CREATED_AT"
+                    ''
+                      https://docs.github.com/en/graphql/reference/enums#issueorderfield.
+                    '';
+                direction =
+                  helpers.defaultNullOpts.mkEnum
+                    [
+                      "DESC"
+                      "ASC"
+                    ]
+                    "DESC"
+                    ''
+                      https://docs.github.com/en/graphql/reference/enums#orderdirection.
+                    '';
+              };
+        };
+  };
+
+  settingsExample = {
+    ssh_aliases = {
+      "github.com-work" = "github.com";
+    };
+
+    # options not defined
+    mappings_disable_default = true;
+    mappings = {
+      issue.react_heart = "<leader>rh";
+      file_panel.select_prev_entry = "[q";
+    };
+    picker = "fzf-lua";
```

Comment:
This line seems to be causing the [CI failure](https://buildbot.nix-community.org/#/builders/165/builds/346) (Error loading picker provider fzf-lua):

```
plugins-git-octo-example> Running phase: patchPhase
plugins-git-octo-example> Running phase: updateAutotoolsGnuConfigScriptsPhase
plugins-git-octo-example> Running phase: configurePhase
plugins-git-octo-example> no configure script, doing nothing
plugins-git-octo-example> Running phase: buildPhase
plugins-git-octo-example> ERROR: Error detected while processing /nix/store/6rj1b1syw12nkgh15g13qqdkivqs1mii-init.lua:
plugins-git-octo-example> Error loading picker provider fzf-lua
plugins-git-octo-example> Cannot request projects v2, missing scope 'read:project'
error: builder for '/nix/store/d83f3vcjkinm2acriiqlkfp1yaizzgak-plugins-git-octo-example.drv' failed with exit code 1
```